packages/flutter/lib/src/material/text_field.dart

```diff
@@ -543,10 +643,21 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
     && widget.decoration != null
     && widget.decoration.counterText == null;
 
-  bool _shouldShowSelectionToolbar = true;
-
   bool _showSelectionHandles = false;
 
+  _TextFieldSelectionHandlerProvider _selectionHandlerProvider;
+
+  // API for EditableTextGestureDetectorClient.
+  @override
+  bool forcePressEnabled;
+
+  @override
+  GlobalKey get editableKey => _editableTextKey;
```

Comment:
Maybe make `editableTextKey` public instead of providing one private and one public?